package.json

```diff
@@ -43,21 +43,21 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@razee/kubernetes-util": "^1.0.0",
-    "@razee/razeedeploy-core": "^1.1.0",
+    "@razee/kubernetes-util": "^1.0.1",
+    "@razee/razeedeploy-core": "^1.1.1",
     "handlebars": "^4.7.7",
     "js-yaml": "^4.1.0",
     "mustache": "^4.2.0",
-    "nopt": "^5.0.0",
+    "nopt": "^6.0.0",
```

Comment:
Are we sure we don't rely on any of the incompatible changes implied by the major version change?

Reply:
not sure, switched version back

Reply:
Guessing you would be ok at 6.0.0 
Release notes say the incompatibility is dropping support for older versions on NodeJS (all of which are out of support)
(https://github.com/npm/nopt/releases)